### ISSUE TITLE
Following redirects drops cookies.

### ISSUE THREAD

**Issue by ixti** [httprb/http#264]
Reported by @oz: https://github.com/httprb/http/issues/263#issuecomment-156260388

Upon following redirects, redirector should send cookies that were sent in response.

**Comment by tarcieri** [comment]
Note that this needs to be done carefully with respect to cookie domain scoping rules or it's a potential security vulnerability (e.g. in conjunction with an open redirect)

**Comment by ixti** [comment]
Good catch. Bright side is that `Cookie::Jar` has that feature already ;)) So I'll just need to make sure to use it :D

**Comment by Kache** [comment]
Hi, looking for some feedback in https://github.com/httprb/http/pull/613 that addresses this issue.

**Comment by ixti** [comment]
@Kache thank you, sorry for being a bit slow on this, will tackle your PR this weekend.

**Comment by tarcieri** [comment]
We have #705 complaining about the opposite problem now, that cookie scoping rules aren't implemented correctly.

Regardless, #613 seems to have implemented this, so closing.
